checkout.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>Tabby Checkout</title>
+
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+      }
+
+      html,
+      body {
+        width: 100%;
+        height: 100%;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
+      }
+
+      #tabby-checkout {
+        width: 100%;
+        height: 100%;
+      }
+
+      .content {
+        padding: 20px;
+      }
+
+      .button {
+        height: 42px;
+        background: #3eedbf;
+        border-radius: 3px;
+        text-transform: uppercase;
+        font-size: 14px;
+        transition: background 0.1s;
+        cursor: pointer;
+        padding: 0;
+        border: 0;
+        outline: 0;
+        width: 100%;
+        margin-bottom: 20px;
+      }
+
+      .control {
+        background: white;
+        border: 1px solid #505050;
+        text-transform: initial;
+      }
+
+      .control {
+        margin-top: 100px;
+      }
+
+      .control + .control {
+        margin-top: 20px;
```

Comment:
@ahmednabil310 I don't understand what this page is. Can you explain?

Reply:
@LiorAgnin sorry this was added by mistake you can delete it